tests/WebAppCallsWebApiCallsGraph/Client/Services/ITodoListService.cs

```diff
@@ -3,7 +3,7 @@
 
 using System.Collections.Generic;
 using System.Threading.Tasks;
-using TodoListService.Models;
+using TodoList.Models;
```

Comment:
I renamed the namespace because:
- it's common to the service and client (it's even the same file with a link)
- There is was a naming conflict between TodoListService (the namespace for this models), and the TodoListService class in the TodoListClient.